smoke/internal/testdata/admin_settings.yml

```diff
@@ -33,6 +33,7 @@ ui:
   url: "http://localhost:9090"
   httplistenaddr: ":9090"
   cors_origins: ["http://localhost:3000"]
+  valid_content_types: ["application/json", "text/plain", "text/csv"]
```

Comment:
maybe a dumb question but what does this do? 🤔

Reply:
+1, it seems like the smoke test is irrelevant to this change.